### PR TITLE
Feature/google drive as plugin

### PR DESCRIPTION
**Pull request by Daboo1998**
Implemented google drive as plugin

### CODE REVIEW

File: packages/plugin_google_drive/package.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "plugin_google_drive",
+  "version": "0.1.0",
+  "private": true,
+  "port": 3002,
```
Comment (dmitry-yudakov) [inline packages/plugin_google_drive/package.json:5]: In future we better remove these ports from here and use `PORT=3002 yarn start`. Port 3001 should currently be passed in the plugin-momentum script.

File: packages/plugin_google_drive/public/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
Comment (dmitry-yudakov) [inline packages/plugin_google_drive/public/index.html:1]: We'll also need to see how to move these to the SDK.

**Review by dmitry-yudakov — APPROVED** [review]
I approve, It's ok considering the current architecture.

If we agree to have a hook like 

```js
const { value: doc, setValue } = useStateItem('doc');
```

it would probably spare us at least one store, or perhaps all of them.

File: packages/plugin_google_drive/package.json (hunk 1 above)
Comment (Daboo1998) [reply packages/plugin_google_drive/package.json:5]: I think we should provide both ways, but rename port to devPort or something like that

**Review by aminbksi — APPROVED** [review]

File: packages/plugin_google_drive/public/index.html (hunk 2 above)
Comment (Daboo1998) [reply packages/plugin_google_drive/public/index.html:1]: idk about that, cause the developer might want to change the index.html (craco or react-script, for example also do not hide it)

**Pull request merged**